flake.nix

```diff
@@ -459,24 +452,84 @@
           };
 
           devShells = rec {
-            developers = mkShell {
+            developers = mkShell (common-attrs // {
               inputsFrom = builtins.attrValues self.checks;
               buildInputs = with packages; [
                 # with nix developers are empowered for local dry run of most ci
-                rust-stable
+                llvmPackages_latest.llvm
+                llvmPackages_latest.bintools
+                lldb
+                zlib.out
+                xorriso
+                grub2
+                qemu
+                llvmPackages_latest.lld
+                openssl.dev
+                pkg-config
+                openssl
+                rust-nightly
```

Comment:
Do we really need all the tooling, can we have a separate shell with rust-nightly only so that we directly hack into the code?

Reply:
Ah